src/main/java/com/littleBeasts/GameLogic.java

```diff
@@ -30,10 +31,11 @@
     public static final Font MENU_FONT = new Font("Serif", Font.BOLD, 13);
     public static String START_LEVEL = "Arkham";
 
-    private static List<Beast> beastList;
-    private Camera camera;
+    private static final List<Beast> beastList = new ArrayList<>();
```

Comment:
What is this List for? and why is it final? A final list can't be changed, which eliminates most of the UCs for a list.

Reply:
To know which beasts are in battle atm

Reply:
the final was added by IntelliJ, I don't question the wisdom of IntelliJ. :-D